### ISSUE TITLE
The contents of array are displayed incorrectly

### ISSUE THREAD

**Issue by FMarks** [cweijan/autohotkey-plus#56]
vscode 1.51.1 / vscode-autohotkey 2.60 / ahk 1.1.33.02

```
#Requires AutoHotkey 1.1.33+

#Warn All
#SingleInstance Force
#NoEnv

SetWorkingDir %A_ScriptDir%
SetBatchLines -1

array := {}

; --- Something strange
; array[99] := "some value"
; array[100] := "now see debugger"

; Debugger show:
; ...
; array: Array(100) <-- ? incorrect array size ?
; [99]: some value
; [100]: now see debugger
; ...

; --- Something strange
array[100] := "some value"
array[101] := "now see debugger"

; Now debugger show:
; ...
; array: Array(101)
; [0..99]
;    [1]: some value
;    [2]: now see debugger
; [100..100]
; ...
```

**Comment by FMarks** [comment]
Can someone confirm or deny it?
Because now i checked on vscode 1.49.3 and above / vscode-autohotkey 2.4.8 and above. And i've been getting this bug everywhere.
But i do remember that before the 1.51 update (about a week ago) everything worked fine.

**Comment by cweijan** [comment]
I no longer develop the debugger function, I suggest you try additional installation [vscode-autohotkey-debug](https://marketplace.visualstudio.com/items?itemName=zero-plusplus.vscode-autohotkey-debug)

**Comment by fade2gray** [comment]
Yes, I can confirm that I see the same behaviour, but I'm a novice regards fully understanding arrays and objects etc.

Sorry, didn't notice cweijan had already replied, but I also see the same behaviour with vscode-autohotkey-debug.
```
array1:={99:"99",100:"100"}
array2:={100:"100",101:"101"}
pause  ; <- set breakpoint here
Exit
```

![AHK-debugger](https://user-images.githubusercontent.com/1332009/99675235-e283f500-2a6e-11eb-8895-ebfd13c4e760.PNG)

![AHK-debugger-2](https://user-images.githubusercontent.com/1332009/99683236-06980400-2a78-11eb-9146-b0dfe0977202.PNG)

**Comment by FMarks** [comment]
**fade2gray** many thanks. I thought i was the only one who have this bug.
**cweijan** I already tried 'vscode-autohotkey-debug' - same bug. Anyway, ty for the great extension.

**Comment by fade2gray** [comment]
@cweijan 

> I no longer develop the debugger function, I suggest you try additional installation vscode-autohotkey-debug

Does this mean you might pull the debugger functionality altogether?

As it is, I can't run  `vscode-autohotkey-debug` whilst using `AutoHotkey Plus` because of the conflicting `configuration type` (autohokey vs ahk) in the `launch.json` file - which you suggested I customize here #40.

**Comment by mark-wiemer** [comment]
Hi @FMarks, if you want this issue fixed in the main extension, please open an issue in [my fork](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus), and I'll look into it! It is published in VS Code as "AutoHotkey Plus Plus", and it includes all the features of this extension, plus some bugfixes and readability enhancements.

**Comment by mark-wiemer** [comment]
![image](https://user-images.githubusercontent.com/7833360/205424157-04284b4e-6100-4fbc-b7f0-0fc3500e8e1f.png)

array: Array(101)
- [0..99]
  - [1]: some value
  - [2]: some value
  - [3]: now see debugger
- [100..100]

Slightly different than described, but still a bug
